Assets/Scripts/Models/Time/TimeManager.cs

```diff
@@ -198,6 +212,15 @@ public void Destroy()
     }
 
     /// <summary>
+    /// Runs the next frame before any updates are ran.
```

Comment:
Well, a quick Google search seems to suggest that "are ran" is never correct, but this is really not the worst thing in comments I have seen since I have returned, so I it's nothing I need to have fixed immediately. Both are understandable, but my bigger issue with that line would be more that I need an extra "in" to parse the sentence as you have it intended:

```csharp
/// <summary>
/// Runs in the next frame before any updates are run.
/// </summary>
``` 
Otherwise it could potentially be parsed as "runs _the next frame_" as in "we do all the updates of next frame _now_". Or maybe that's just my caffeinated, insomniac brain parsing English weirdly.